### DISCUSSION TITLE
Railway部署服务端后不知道如何添加客户端

### DISCUSSION THREAD

**Discussion by DifferentSparkler** [zdz/ServerStatus-Rust#39] [Q&A] [answered]
对于我这种小白来说看了好久的文档还是看不懂
建议Railway部署教程后面增加一个完整添加客户端的图文教程
服务端应该如何操作  需要改哪些参数   客户端应该如何操作  需要修改哪些参数

![image](https://user-images.githubusercontent.com/38945550/168637382-42721b75-119e-4b9e-858a-69258d51bedd.png)

**Comment by zdz** [comment, marked as answer]
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/152173/168640176-6290bea5-fc30-4014-9e3e-2d102c58e7f4.png">
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/152173/168640230-40952d6c-0378-4566-997c-96a573601b01.png">

**Comment by zdz** [comment]
1. 代码做好排版 **（ 排版，排版，排版）**，不要随便往上一粘就完事
2. 侦听地址用默认的就行
3. 报错了不是应该贴报错信息？

**Reply by DifferentSparkler** [reply]
> 1. 代码做好排版 **（ 排版，排版，排版）**，不要随便往上一粘就完事
> 2. 侦听地址用默认的就行
> 3. 报错了不是应该贴报错信息？

请教下railway的服务端是不是只用更改
（hosts = [
  {name = "h1", password = "p1", alias = "woiden1", location = "🏠", type = "kvm", notify = true},
  {name = "h2", password = "p2", alias = "woiden2", location = "🏢", type = "kvm", disabled = false},
  {name = "h3", password = "p3", alias = "n3", location = "🏝️", type = "kvm", monthstart = 1},
]）
这一部分实现添加客户端


客户端是不是需要更改
（http://127.0.0.1:8080/report）
这一部分为railway的应用地址
（https://serverstatus-rust-production-***.up.railway.app）
不知道我这样理解对不对 


能不能把你演示地址（https://tz-rust.vercel.app）的服务端  客户端配置文件样板上传分享一下

**Reply by zdz** [reply]
是的，config.toml 文件就是 server 的样板，客户端就是教程文档加 -u -p 参数指定机器就能跑，不需要样板。你都能成功加上第一个 client 了，其它不是类似的么
